packages/graphql/src/classes/NodeDirective.ts

```diff
@@ -48,9 +51,16 @@ export class NodeDirective {
     }
 
     public getLabels(typeName: string, context: Context): string[] {
-        const mainLabel = this.label || typeName;
-        const labels = [mainLabel, ...this.additionalLabels];
-        return this.mapLabelsWithContext(labels, context);
+        let finalLabels: string[] = [];
```

Comment:
`finalLabels` feels a bit confusing!

```suggestion
        let labels: string[] = [];
```